examples/mongodb/1.8/initialize-mongo-single.js

```diff
@@ -0,0 +1,83 @@
+let error = true
+
+let res = [
+// remove if exists
+ db = db.getSiblingDB('inventory'),
+ db.customers.drop(),
+ db.orders.drop(),
+ db.products.drop(),
+
+  rs.initiate({
```

Comment:
is this really working with Mongo DB 5 now? Previously it was necessary to initialize a replica set separately from another init container process because the init scripts were executed in a special "setup/standalone mode" where functionality like replica sets aren't available.
Compare with that comment in docker-entrypoint.sh 
https://github.com/docker-library/mongo/blob/master/5.0/docker-entrypoint.sh#L299